md/bpm-api.md

```diff
@@ -481,6 +481,18 @@ Retrieve humanTask objects that match the specified filters.
   `GET`
 * **Data Params**  
   [Standard search parameters](rest-api-overview.md#resource_search) are available.  
+  
+  * o: only parameters available in HumanTaskInstanceSearchDescriptor.java and in the result are accepted: 
+    * assignee_id
+	* diplayname
```

Comment:
good question, usually, we configure Eclipse to have `tab` key converted to spaces (4 generally)